index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <style>
+    #navbar {
+      align-items:left;
+      position:fixed
+    }
+    @media only screen and (max-width: 800px) {
+      [class*="col-"] {
+        width: 100%;
+      }
+    }
+  </style>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+    <main id="main-doc">
+      <nav id=navbar>
+        <header>
+          Coding
+         </header>
+         <a class="nav-link" href="#introduction">Introduction</a>
+          <a class="nav-link" href="#what_you_should_already_know">what you should already know</a>
+          <a class="nav-link" href="#javascript_and_java">javascript and java</a>
+          <a class="nav-link" href="#hello_world">hello world</a>
+          <a class="nav-link" href="#variables">variables</a>
+      </nav>
+      <section id="introduction" class="main-section">
+        <header id="introduction">
```

Comment:
Id's should be unique, try removing this item's id or its parent, either way, you only need one :wink: 

Also, a header tag is usually used in the header element, you should use a `<div>` to follow best practices!

Reply:
- [ ] I think it would be better if you use suggestive phrases instead of using words that read as an order when requesting required changes so that the student gives emphasis to changes requested.